src/LaravelModulesServiceProvider.php

```diff
@@ -74,4 +77,27 @@ protected function registerServices()
         });
         $this->app->alias(Contracts\RepositoryInterface::class, 'modules');
     }
+
+    protected function registerMigrations(): void
+    {
+        if (! $this->app['config']->get('modules.auto-discover.migrations', true)) {
+            return;
+        }
+
+        $this->app->resolving(Migrator::class, function (Migrator $migrator) {
+
+            $path = implode(DIRECTORY_SEPARATOR, [
+                $this->app['config']->get('modules.paths.modules'),
+                '*',
+                '[Dd]atabase',
```

Comment:
Is this correct?

Reply:
The old module database directory is named `Database` with an uppercase "D", while in the new structure, it is lowercase `database`.

The `glob` function reads directories using this regex pattern.